app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -51,14 +52,84 @@ export const translateWord = (operand, word, translations) =>
     ? word
     : I18n.t(`story.${operand}.${translations[word] || word}`, { locale: I18n.defaultLocale });
 
-const translatedOperands = ['type','state'];
+const translatedOperands = ['type', 'state'];
 
 export const haveTranslation = operand => translatedOperands.includes(operand)
 
 export const isEnglishLocale = () => currentLocale() === 'en';
 
 const currentLocale = () => I18n.currentLocale();
 
+// Drag And drop utils
+
+const calculatePosition = (aboveStory, bellowStory) => {
+  if (bellowStory === undefined) return Number(aboveStory.position) + 1;
+  if (aboveStory === undefined) return Number(bellowStory.position) - 1;
+  return (Number(bellowStory.position) + Number(aboveStory.position)) / 2;
+};
+
+export const getNewPosition = (
+  destinationIndex,
+  sourceIndex,
+  storiesArray,
+  isSameColumn,
+  storyType,
+) => {
+  if(storiesArray.length === 0){
```

Comment:
You can use [`isEmpty`](https://underscorejs.org/#isEmpty) from `underscore` here